src/include/daos_obj.h

```diff
@@ -220,9 +220,9 @@ static inline bool
 daos_is_array_type(enum daos_otype_t type)
 {
 	switch (type) {
+	case DAOS_OT_ARRAY_BYTE:
```

Comment:
I can probably revert these, just figured it might be better to put the more likely one on top